osgp/shared/shared/src/main/java/org/opensmartgridplatform/shared/infra/jms/ResponseMessage.java

```diff
@@ -58,7 +58,8 @@ private String serializableToString(final Serializable dataObject) {
             return "";
         } else {
             final String stringValue = dataObject.toString();
-            return stringValue.substring(0, Math.min(stringValue.length(), 40));
+            //TODO temp change; reset to 40
+            return stringValue.substring(0, Math.min(stringValue.length(), 128));
```

Comment:
If the message in the TODO still applies, change the length back to a maximum of 40,
Anyway, the TODO should not be left in the code.